rcl_interfaces/msg/ParameterDescriptor.msg

```diff
@@ -19,6 +19,7 @@ string description
 string additional_constraints
 
 # If 'true' then the value cannot change after it has been initialized.
+# But parameter still can be undeclared with force flag.
```

Comment:
I agree on having a comment here, but the comment should mention that the force flag can be used only by the node declaringthe parameter.

Something like
```
# But parameter still can be undeclared by the node that declared it with the force flag.
```